### PR TITLE
caf 0.17.4

### PR DESCRIPTION
**Pull request by Neverlord**
Created with `brew bump-formula-pr`.

---

Update CAF to version 0.17.4

### CODE REVIEW

**Comment by bayandin** [comment]
Formula `zeek` requires a version bump.

And
```
Error Message
failed: brew audit caf --online
Stacktrace
        Error: 1 problem in 1 formula detected
caf:
  * C: 23: col 5: Formulae in homebrew/core (except e.g. cryptography, libraries) should not run build-time checks
```

**Comment by Neverlord** [comment]
> Formula `zeek` requires a version bump.

I'm not sure what step is needed here. The formulae for Zeek [does not ask for a specific CAF version](https://github.com/Homebrew/homebrew-core/blob/master/Formula/zeek.rb#L18):

```ruby
  depends_on "caf"
```

File: Formula/caf.rb [hunk 1]
```diff
@@ -14,7 +13,7 @@ class Caf < Formula
     sha256 "4a0c871ff5848389f01c21d85a6d9b59c6b5d73404329bfa408b732d62b5a2bf" => :high_sierra
   end
 
-  depends_on "cmake" => :build
+  depends_on "cmake"
```
Comment (bayandin) [inline Formula/caf.rb]: `cmake` requires only for building, so revert these changes back, please

**Review by bayandin — COMMENTED** [review]
And linter complains about`system "make", "--directory=build", "test"` line

**Comment by bayandin** [comment]
Regarding `zeek`, because `zeek` it's built and linked to the old (the current one) version of `caf` we need to rebuild this formula with a new one. To do it we use revisions. The easiest way to bump it is run a command `brew bump-revision --message="for caf" zeek` and this will prepare a commit for you, and you need just push it.

**Comment by Neverlord** [comment]
Thanks for the explanations. I hope I got it right this time. 🙂

**Comment by bayandin** [comment]
Well done @Neverlord, without people like you, we couldn't run this project! 🎉
